pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml

```
@@ -24,7 +24,20 @@ spec:
       description: The workspace containing the function project
   steps:
   - name: func-deploy
-    image: "registry.redhat.io/openshift-serverless-1/kn-plugin-func-rhel8:latest"
+    image: "registry.access.redhat.com/ubi8/ubi"
     script: |
+      arch="amd64"
```

```suggestion
      set -Eeuo pipefail
      arch="amd64"
```

See: http://redsymbol.net/articles/unofficial-bash-strict-mode

We use pipes in the script, so definitely the `pipeline` is required. The `-E` and `-e` options ensures error will end the script. The `-u` fail in case a unknown variable is referenced.